source/_components/sensor.fronius.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: page
+title: "Fronius "
+description: "Instructions on how to connect your Fronius Inverter to Home Assistant."
+date: 2018-01-05 11:43:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Energy
+logo: fronius.png
+ha_iot_class: "Local Polling"
+ha_release: 0.60
+---
+
+
+The `fronius` sensor will poll a [Fronius](http://www.fronius.com/) solar inverter, battery system or smart meter and present the values as sensors (or attributes of sensors) in Home Assistant.
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: fronius
+    host: 192.168.42.47
+    device: 1
+    type: inverter
+
+  - platform: fronius
+    host: 192.168.42.47
+    device: 2
+    type: inverter
+
+  - platform: fronius
+    host: 192.168.42.47
+    type: inverter
+    scope: system
+
+  - platform: fronius
+    host: 192.168.42.49
+    type: storage
+
+  - platform: fronius
+    host: 192.168.42.49
+    type: meter
+
+  - platform: fronius
+    host: 192.168.42.49
+    type: power_flow
+
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration